test/manifests.go

```diff
@@ -173,7 +182,8 @@ var IntegrationManifestJSON string = `{
 					"ROOT_CA": "{{ pem .Marblerun.RootCA.Public }}",
 					"SEAL_KEY": "{{ hex .Marblerun.SealKey }}",
 					"MARBLE_CERT": "{{ pem .Marblerun.MarbleCert.Public }}",
-					"MARBLE_KEY": "{{ pem .Marblerun.MarbleCert.Private }}"
+					"MARBLE_KEY": "{{ pem .Marblerun.MarbleCert.Private }}",
+					"TEST_SECRET_HEX": "{{ hex .Secrets.testsecret_raw }}"
```

Comment:
```suggestion
					"MARBLE_KEY": "{{ pem .Marblerun.MarbleCert.Private }}"
```
I think we don't need this in the integration test. We should limit the integration test to parts that can't be covered with unit tests.